project_report/tex/analysis.tex

```diff
@@ -15,3 +15,9 @@ \subsubsection{Sequence Diagrams}
 \subsubsection{Operation Contracts}
 
 \subsubsection{Updated UML Class Diagram}
```

Comment:
Write a description including:
  - What are we seeing here?
  - What can this be used to in our development?